csrc/scheduler/reduction_utils.cpp

```diff
@@ -1009,5 +1003,30 @@ std::ostream& operator<<(std::ostream& os, ReductionType reduction_type) {
   return os;
 }
 
+void sharedMemoryConsumerVectorization(
+    std::vector<TensorView*>& smem_consumers) {
+  // Vectorization of smem consumers, they were created with cacheAfter().
```

Comment:
This comment seems to largely overlap with the comment of the header file. Have some comment that's important for the user of this function in the header. The implementation file should be mostly about implementation details.

Looks like both the header file and this file only discusses implementation details. See my comment at the header file as well.

Reply:
Thanks for the suggestion. Revised and proposed to follow the same style in the future.
For declaration in header file:
1. Use Doxygen-style comments before the function declaration.
2. Focus on describing the interface - what the function does, parameters, return value.
For implemenation in cpp file:
1. explaining the implementation details focus on `why` if `what` is simple and clear.